diesel_codegen/src/schema_inference/mod.rs

```diff
@@ -1,7 +1,12 @@
 mod data_structures;
+#[cfg(feature = "postgres")]
+#[path="pg.rs"]
+mod implementation;
+#[cfg(feature = "sqlite")]
```

Comment:
We need to support using both PG and SQLite in the same code base. We should do the same thing that we do in CLI here.